FirebaseDynamicLinks/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# v8.10.0
```

Comment:
Next release is 8.15.0. See https://github.com/firebase/firebase-ios-sdk/milestones

Reply:
Done